guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
```

Comment:
<p>For <strong>multi-label classification</strong>, we simply need to pass <code>multi_class=True</code> in the <code>predict()</code> method of the model. The <strong>resulting per label</strong> scores for multi-label classification are independent <strong>probabilities </strong>and fall in <strong>the </strong>(0,1) range.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>fixed</p>

Reply:
<p>Hi Anurag, Sorry for the confusion, i did not mean for "probability" and "(0,1)" to be in bold letters. I was trying to highlight the change by putting it in bold. </p>

Reply:
fixed